Libraries/Opc.Ua.Gds.Server.Common/ApplicationsNodeManager.cs

```diff
@@ -516,7 +490,7 @@ out nextRecordId
             ApplicationRecordDataType application,
             ref NodeId applicationId)
         {
-            HasApplicationAdminAccess(context);
+            AuthorizationHelper.HasAuthorization(context, new List<Role> { GdsRole.DiscoveryAdmin });
 
```

Comment:
nit: could the code be a little cleaner if the role lists were predefined as static readonly/readonly list or so?

Reply:
done, good idea